aries-site/src/pages/components/button.mdx

```diff
@@ -54,6 +54,18 @@ Using the design system Button Component will ensure your button looks like a bu
 - Avoid contractions or colloquialisms. Keep the language approachable but not personal, for examples, see [point of view](https://design-system.hpe.design/foundation/voice-and-tone?q=tone#point-of-view).
 - Ensure your button label matches the route destination.
 
+#### Guidance on when to use 'add', 'create' and 'new'.
+
+**Add**: ‘Add’ should be used where someone can add something that is already existing. It’s recommended to include the name of the thing that user can create. Combine add with the object where appropriate, Add+thing to create. 
+Examples: Add user, Add file, or Add member. 
+
+**New**: Use ‘New’ for creating something new, a new object Include the name of the thing that user can create.
```

Comment:
```suggestion
**New**: Use ‘New’ for creating something new, a new object. Include the name of the thing that user can create.
```

Reply:
I'm not following the nuance of "New" here vs "Add". Is there anything we can add to be clearer about why I should do "New account" as opposed to "Add account"? @SeamusLeonardHPE @ashifalinadaf

Reply:
Discussion on the benefits and drawbacks of allowing "New" discussed in the ticket thread in this comment and the comment below:
https://github.com/grommet/hpe-design-system/issues/3496#issuecomment-1699467337

Reply:
There are certain words we use like 'new tab 'rather than create tab, in such cases its recommended to use the new instead of create. 

comments from Scott on the same, \

> 'create tab' doesn't seem to have the concise label compared to 'new tab'. It isn't a verb but I is still clear and familiar.

Reply:
I understand that use cases warrant it, but what I'm trying to say is that I don't feel the written guidance is clear enough for designers to know when to use "New" vs "Create".

Any wording suggestions? Rationale we can add? Do/dont? Not sure the best mechanism